### DISCUSSION TITLE
The surprising bitrates abilities of HERO13

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#1095] [General]
I was playing with the new 360fps video burst, combined with Ultra Wide for pretty nice results.  It set the bit-rate in-camera to High, results are at around 120Mb/s.  Yet when this is played at 24fps (15x slower,) that is only 8Mb/s at playback speed, which can be improved via Labs.  HERO12 was capped out at 200Mb/s, and mostly reliable at 180Mb/s or less. For video burst slow-mo, my HERO13 captured at over 600Mb/s. Just WOW!!!  For regular video mode 200Mb/s now seems easy, 300Mb/s is possible on this HERO13.  I was testing with firmware 1.12.70 and 256GB SanDisk Extreme Pro cards.  Anyone finding this too?

**Comment by RikshaDriver** [comment]
Interesting. On my H13, I've managed to get sustained 300Mbps for 4K 100p on a Sandisk Extreme card. Tried 450Mbps but it conked out on me after nearly 2 minutes. I set 400Mbps, but it seems to have maxed out around 300Mbps and recorded about 38 minutes worth until it stopped and complained. Might need to find a way to push the codec as it's likely finding efficiencies to reduce the bitrate.

BTW, I tested the card using H2testw and was working at a sustained ~75MB/s (~600Mbps) transfer rate until completely full

**Comment by dnewman-gpsw** [comment]
Posts seem to be disappearing, please don't delete posts, as there are always useful to help others.

**Comment by Rabii-Hayani** [comment]
All this capabilities with hero 13 that can pump up to 300 mbs should be combined with a great profesionel codec like Cinform or somthing like that to geat the best quality from it

**Reply by dnewman-gpsw** [reply]
There is no hardware acceleration for CineForm. The raw photos (.GPR) are CineForm SMPTE VC5, and the camera's ARM cores can only encode a frame every 1-2 seconds. Bit-rate is less of an issue, compute power is reason. But even if the compute was available, CineForm is an I-frame codec, so to encode with its full quality, the raw sensor data (CineForm Raw) requires a bitrate closer to 3Gb/s (about 10X your SD can do) and that is only for 24p. Doing only 3840x2160 4:2:2 would help, but 4k60 is still ~1200Mb/s. As the original developer of CineForm, I know it has its place, but long GOP compression (like 10-bit HEVC) with sufficient bit-rate, is a high quality capture solution.
